kotlin/src/main/com/looker/sdk/3.1/methods.kt

```diff
@@ -3929,6 +3934,30 @@ class Looker31SDK(authSession: AuthSession) : APIMethods(authSession) {
 
     //region LookmlModel: Manage LookML Models
 
+    /**
+     * ### Discover information about derived tables
+     *
+     * @param {String} model The name of the Lookml model.
+     * @param {String} format The format of the graph. Valid values are [dot]. Default is `dot`
+     * @param {String} color Color denoting the build status of the graph. Grey = not built, green = built, yellow = building, red = error.
+     *
+     * GET /derived_table/graph/model/{model} -> DependencyGraph
+     */
+    @JvmOverloads fun graph_derived_tables_for_model(
```

Comment:
maybe you generated against master instead of 7.20? I think this is in 21.0 (MOD-2999)

Reply:
yup. regenerated now.